src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -28,6 +29,18 @@ class AboutCommand extends ContainerAwareCommand
 {
     /**
      * {@inheritdoc}
+     *
+     * @deprecated since version 3.4, to be removed in 4.0
+     */
+    protected function getContainer()
```

Comment:
But adding `since version 3.4` in 3.4 branch doesnt make sense right? Or do you sugges to patch 3.3 first?